framework/components/ecl-carousels/ecl-carousels.scss

```diff
@@ -0,0 +1,120 @@
+/**
+ * Carousel
+ * @define carousel ; weak
+ */
+
+.ecl-carousel {
+  align-items: stretch;
+  background-color: #000;
+  display: flex;
+  flex-direction: column;
+  margin: 0;
```

Comment:
Honestly, I tried, but the result was still different and not very likable - i.e. the content in the modal doesn't get fully displayed. I will try further.